### ISSUE TITLE
"Admin" login not working after update to 1.34

### ISSUE THREAD

**Issue by leveraged7** [causefx/Organizr#332]
Never REALLY had an issue with Organizr the entire time using this wonderful creation (other than my own config setting for headphones.)

But since performing the auto update to 1.34,  I can't login using the local "Admin" account for Organizr. I have to login using one of the Plex logins. Luckily one of the logins has Admin rights, so can make changes, etc., BUT when I go into the log section, it shows the bad logs ins, the local Admin username login is in all lowercase. example "localadminusername" when in reality, I entered in "LocalAdminUsername" which if I goto the usertabs, shows up as "LocalAdminUsername".

I am running Windows 10, with Nginx, and haven't made ANY other changes other than the forced update from Win10 AND the Organizr Update to 1.34. 

Thank you once again!

**Comment by Cerothen** [comment]
Please swap out your "user.php" for this updated one then try to log in with the account your trying to use.
https://github.com/causefx/Organizr/blob/cero-dev/user.php
https://github.com/causefx/Organizr/commit/1b91dec8bb405c774769e0176738deddd846dd3b
If you could report back success or not that would be ideal. Thanks,

**Comment by leveraged7** [comment]
Doing it now. I will let you know in a few moments. AT work so like, I have to be stealthy about this.
Also, are there two files there or just the one? (user.php)  there is a link below it, not sure what that is.(causefx/organizr@1b91dec)

**Comment by Cerothen** [comment]
One link is just the git diff, there is only one file

**Comment by leveraged7** [comment]
OH okay. Thank You. That's a no go. It did not work. I even stopped and restarted both the php server and nginx.
However, it is still showing the localadmin user account as logged in. I am not sure if that means anything or not.

**Comment by Cerothen** [comment]
Bummer, alrighty. I will have to try something else

**Comment by leveraged7** [comment]
Thanks for your help once again! BTW, since I am a belt and suspenders kind of guy. I downloaded the Cero-Dev zipfile. unblocked it (I have win10), then extracted it to the download directory.  Then stopped both nginx and phpserver. Then renamed the previous user.php file to olduser.php. then copied over the user.php from the extracted & downloaded  cero-dev folder. Restarted nginx, php.server. then opened cmd prompt, and reload nginx, nginx -s reload  just to make sure adn then finally attempted login.

**Comment by leveraged7** [comment]
#276 & #298 
These two might also be related. Just an FYI, I am getting the same errors as I have been messing with it.

**Comment by MartinAyla** [comment]
I'm locked out as well, after updating Organizr.

I only had my local admin account, so I can't login at all now.

Can I manually add a user in a conf file or something?

Thanks

**Comment by Cerothen** [comment]
@leveraged7 Currently the settings are split up tabs and users are in the database file, basically every other setting is in the config.php file. Deleting your database file would solve most things but you would need to setup the users and tabs. What happens if you delete the user admin you cant log in as and remake it?

@MartinAyla There isn't to much you can do manually short of deleting the users.db file and readding the tabs/users. Did your username also have a capital in it? I am trying to hammer down what is the root cause of the issue.

**Comment by MartinAyla** [comment]
@Cerothen Thanks for the reply. Yes my username has a capital in it. And my password is long, with special characters in it.

Should I wait a little while to see if there's a fix, or will I have to start over (delete users.db) no matter what (if I could avoid setting up all my tabs it would be nice)?

**Comment by Cerothen** [comment]
I am still looking into the issue. Unfortunately I had some pressing items at work and havent had the spare time for a little while.

**Comment by kernelkaribou** [comment]
Hey @Cerothen I am running into the same issue and figured I would provide some of my findings to see if it helps. I am going to include my credentials (random generated and unique so no concern)

User: Yoda
PW: 3G2fdk8DV4gNuTeP

So capital in the user but no special characters for the password.

I noticed that the hash for my user on 1.33 was different than after the update to 1.34

1.33
username    password                                                
----------  ----------------------------------------------------------------  
Yoda        88f5a1e3be736c530b95fc70c07ddf9aad7ac877f895e27284f24012135619f4  

1.34

id          username    password 
----------  ----------  ----------------------------------------------------------------  
1           Yoda        31ed1b0fae758b0ba64c882e8a44003193b9f47ae447c95744d246841c52742d

So not sure if that caused the issue, I tried updating directly in the users.db to have the same hash before and the username to be lowercase as well but that did not work.  I did not look any deeper but was hoping this may help some.  Thanks
